components/renku_data_services/session/api.spec.yaml

```diff
@@ -437,7 +531,7 @@ components:
           name: Rstudio
           creation_date: "2023-11-01T17:32:28Z"
           description: JupyterLab session environment
-          environment_kind: GLOBAL
+          environment_kind: global
```

Comment:
Suggestion: Instead of fixing this, it is preferable to NOT provide examples for objects. Examples are derived by swagger from single field examples and as such are easier to keep well maintained.